Scenes/HUD/ElementTowersMenu.gd

```diff
@@ -194,6 +192,17 @@ func _update_info_label():
 	_element_info_label.text = text
 
 
+func _update_empty_slots():
```

Comment:
This function is duplicated here and in item stash. Make it into a static function and call it from both places:

`static func update_empty_slots_generic(empty_slots: Array, element_list: Array)`

Reply:
I have joined them into UnitButtonsContainer script.